src/main/java/org/opentripplanner/raptor/api/model/RaptorAccessEgress.java

```diff
@@ -45,6 +45,39 @@ public interface RaptorAccessEgress {
    */
   int durationInSeconds();
 
+  /**
+   * Raptor can add an optional time-penalty to a access/egress to make it less favourable compared
+   * with other access/egress/transit options (paths). The penalty is a virtual extra duration of
+   * time added inside Raptor when comparing time. The penalty does not propagate the c1 or c2 cost
+   * values. This feature is useful when you want to limit the access/egress and the access/egress
+   * is FASTER than the preferred option.
+   * <p>
+   * For example, for Park&Ride, driving all the way to the
+   * destination is very often the best option when looking at the time criteria. When an
+   * increasing time-penalty is applied to access/egress with driving then driving less become
+   * more favorable. This also improves performance, since we usually add a very high cost to
```

Comment:
```suggestion
   * increasing time-penalty is applied to a car access/egress, then driving become less
   * favorable. This also improves performance, since we usually add a very high cost to
```